packages/devtools_app/lib/src/shared/dialogs.dart

```diff
@@ -64,10 +70,10 @@ class UnexpectedErrorDialog extends StatelessWidget {
           ),
         ),
         DialogTextButton(
-          child: const Text('Create Issue'),
+          child: const Text('Create issue'),
           onPressed: () => unawaited(
             launchUrl(
-              devToolsExtensionPoints.issueTrackerLink().url,
+              newDevToolsIssueUri(issueDetails: errorDetails).toString(),
```

Comment:
we should use `devToolsExtensionPoints.issueTrackerLink()`, otherwise this will take g3 users to github, which we don't necessarily want.